include/ofi_iov.h

```diff
@@ -99,6 +99,17 @@ ofi_copy_from_iov(void *buf, uint64_t bufsize,
 	}
 }
 
+static inline void
+ofi_copy_iov(struct iovec *iov_dest, const struct iovec *iov_src, size_t count)
+{
+	size_t i;
+
+	for (i = 0; i < count; i++) {
+		iov_dest[i].iov_base = iov_src[i].iov_base;
+		iov_dest[i].iov_len = iov_src[i].iov_len;
+	}
+}
```

Comment:
This should just use memcpy and not iterate over the iov.  I would drop this helper

Reply:
Yes, this can just be a memcpy.